java/kotlin-extractor/src/main/kotlin/comments/CommentExtractor.kt

```diff
@@ -35,7 +35,13 @@ open class CommentExtractor(protected val fileExtractor: KotlinFileExtractor, pr
             tw.getExistingLabelFor<DbTop>(label)
         }
         if (existingLabel == null) {
-            logger.warn("Couldn't get existing label for $label")
+            // Sometimes we don't extract elements.
+            // The actual extractor logic is a bit more nuanced than
+            // just "isFake", but just checking isFake is good enough
+            // to not bother with a warning.
+            if(element !is IrDeclarationWithVisibility || !fileExtractor.isFake(element)) {
```

Comment:
What's your preferred IDE? Is there a way to autoformat code on file save? There's a missing space in `if(`.